### DISCUSSION TITLE
Possible to determine what agent a user is playing with?

### DISCUSSION THREAD

**Discussion by captainswain** [saul/demofile#287] [Questions] [answered]
Does anyone know if it is possible to retrieve what agent skins a player is using from the demo?

**Comment by saul** [comment, marked as answer]
Have you tried `Player#modelName`?

**Reply by captainswain** [reply]
You're a GOAT. I wasn't reading properly as the path had "legacy" in them and assumed it was just pulling the original models pre-agent update. Thanks mate!
